core/templates/domain/topic_viewer/topic-viewer-backend-api.service.spec.ts

```diff
@@ -44,6 +45,7 @@ describe('Topic viewer backend API service', () => {
       'ShortSkillSummaryObjectFactory', new ShortSkillSummaryObjectFactory());
     $provide.value(
       'ReadOnlyObjectFactory', new ReadOnlyTopicObjectFactory(
+        new StoryNodeObjectFactory(),
```

Comment:
Can you replace lines 43-58 with one function call `importAllAngularServices();`. You will have to import the function too: `import { importAllAngularServices } from 'tests/unit-test-utils';` You can also remove remove lines 61-63.